@here/harp-mapview/lib/MapView.ts

```diff
@@ -2914,8 +2927,7 @@ export class MapView extends THREE.EventDispatcher {
                 // also the same. The position is then calculated based on the light direction and
                 // the height
                 // using basic trigonometry.
-                const tilt = this.m_pitch;
-                const cameraHeight = this.targetDistance * Math.cos(tilt);
+                const cameraHeight = this.targetDistance * Math.cos(this.m_viewState.pitch);
```

Comment:
Shouldn't this be computed with `projection.getGroundDistance(m_viewState.worldCenter)` instead? Using trigonometry only works for mercator projection.

Reply:
true. Not sure what was the original intent of this code.